src/reducers/sort.reducer.spec.js

```diff
@@ -0,0 +1,28 @@
+import sortReducer from './sort';
+import constants from '../constants';
+
+describe('[ Reducers ] Sort', () => {
+  let state;
+  beforeEach(() => { state = constants.NO_SORT; });
+
+  it('[INITIAL] should return same state', () => {
+    const action = { type: '@@INIT' };
+    expect(sortReducer(state, action)).toEqual(state);
```

Comment:
You can use ```toBe``` instead of ```toEqual```. toEqual is used for deep comparing and we need it only for objects. For primitive type we can use just toBe